### PR TITLE
Add accuracy disclaimer to test results

### PR DESCRIPTION
**Pull request by FWDekker**
Adds a disclaimer that indicates that the results of Schaapi may not be accurate.

### CODE REVIEW

File: modules/validation-pipeline/github-interactor/src/main/kotlin/org/cafejojo/schaapi/validationpipeline/githubinteractor/CIJobCompletedListener.kt [hunk 1]
```diff
@@ -52,6 +52,10 @@ class CIJobCompletedListener(private val checkReporter: CheckReporter) {
 
     private fun formatTestResults(testResults: TestResults) = with(testResults) {
         """
+            <span style="color:red;">
+                Schaapi is an experimental tool and the accuracy of its results is not guaranteed.
+            </span>
```
Comment (gandreadis) [inline modules/validation-pipeline/github-interactor/src/main/kotlin/org/cafejojo/schaapi/validationpipeline/githubinteractor/CIJobCompletedListener.kt]: I don't think this works in PR reports

Comment (FWDekker) [reply modules/validation-pipeline/github-interactor/src/main/kotlin/org/cafejojo/schaapi/validationpipeline/githubinteractor/CIJobCompletedListener.kt]: :'(
Any suggestions?

Comment (casperboone) [reply modules/validation-pipeline/github-interactor/src/main/kotlin/org/cafejojo/schaapi/validationpipeline/githubinteractor/CIJobCompletedListener.kt]: The only way to use color is by using image. I don't think that's a very good idea 😅.

Therefore I think we can just go with 

```
Schaapi is an experimental tool and the accuracy of its results is not guaranteed.
```

Comment (FWDekker) [reply modules/validation-pipeline/github-interactor/src/main/kotlin/org/cafejojo/schaapi/validationpipeline/githubinteractor/CIJobCompletedListener.kt]: Will do

**Pull request merged**